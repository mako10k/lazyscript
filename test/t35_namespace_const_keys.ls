!{
  ~ns <- ((~prelude .nsnew0));
  ~~nsdefv ~ns .k 1;
  ~~nsdefv ~ns "s" 2;
};
