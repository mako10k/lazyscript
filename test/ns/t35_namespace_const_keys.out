E: namespace: key must be symbol
1

()
